### PR TITLE
[TEAM D] [SQTA-20] As a user, I can view the list of owners.

### PR DESCRIPTION
**Pull request by dylanpita**
Testing the listing of owners.

### CODE REVIEW

**Review by krocden — APPROVED** [review]

File: Team-D/src/test/java/OwnerSeleniumTests.java [hunk 1]
```diff
@@ -1,20 +1,84 @@
...
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
 
 @ExtendWith(SeleniumExtension.class)
 public class OwnerSeleniumTests {
 
     ChromeDriver driver;
 
+    @MockBean
```
Comment (chillman12) [inline Team-D/src/test/java/OwnerSeleniumTests.java]: Delete this.

**Review by mrwompwomp — APPROVED** [review]
looks good!

**Review by mrwompwomp — APPROVED** [review]

**Comment by karimibrahim07** [comment]
Your code looks great, However please remove the @mockbean tag.

**Review by karimibrahim07 — APPROVED** [review]

**Review by namisabet — APPROVED** [review]
Good job

**Pull request merged**